view/Response.class.php

```diff
@@ -106,11 +106,31 @@ public static function addJsAsset($src)
     static::$assets['js'][$src] = $src;
   }
 
+  public static function addCssAsset($src)
+  {
+    static::$assets['css'][$src] = $src;
+  }
+
   public static function getJsAssets()
   {
     return static::$assets['js'];
   }
 
+  public static function getCssAssets()
+  {
+    return static::$assets['css'];
+  }
+
+  public static function setAllCss()
```

Comment:
I think this is the wrong approach. Rather than calling this everywhere, put `all.css` in the default asset definition (similar to JS). Then, add a method called `setCssAssets(array $assets = [])`.

When a page such as yt2 wants to disable use of all.css, it would call `Response::setCssAssets(['css/yt2.css'])`, which would remove the all default.